lib/rules/no-hooks-from-ancestor-modules.js

```diff
@@ -120,17 +120,19 @@ module.exports = {
                     if (node.arguments.length === 0) {
                         return;
                     }
-                    if (
-                        node.arguments[0].type !== "Literal" ||
-                        node.arguments[0].value === null ||
-                        node.arguments[0].value === undefined
-                    ) {
-                        return;
+
+                    let description;
+                    if (node.arguments[0].type === "Literal") {
+                        description = node.arguments[0].value.toString();
+                    } else {
+                        const sourceCode = context.getSourceCode();
+                        description = sourceCode.getText(node.arguments[0]);
                     }
```

Comment:
Does this help? Ternary would be nice.

```suggestion
                    const sourceCode = context.getSourceCode();
                    const arg = node.arguments[0];
                    const description =
                        arg.type === "Literal" && typeof arg.value === "string"
                            ? arg.value.toString()
                            : sourceCode.getText(arg);
```